src/app/spec/component-fixture-spec.helper.ts

```diff
@@ -0,0 +1,119 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {
+    ChangeDetectorRef, DebugElement, SchemaMetadata, Type, Component, ViewChild,
+    ViewContainerRef, ComponentFactoryResolver, Optional, NgModule
+} from '@angular/core';
+import {SelectSpecHelper} from './select-spec.helper';
+import {Predicate} from '@angular/core/src/debug/debug_node';
+import {BrowserDynamicTestingModule} from '@angular/platform-browser-dynamic/testing';
+
+@Component({
```

Comment:
Maybe you can add a comment with an example on how it is used.

Reply:
This is still open I guess, does not necessarily need to be much detailed.

Reply:
The TestHostComponent is not visible for the user. A comment on how to use the Helper Class is over the declaration of the Helper class. I can write a short comment on what the TestHostComponent is used for.